service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersUploadedEventHandler.java

```diff
@@ -44,39 +51,87 @@ public void sendNotificationToJudge(final DraftOrdersUploaded event) {
             .map(Element::getValue)
             .orElse(null);
 
-        final List<HearingOrder> orders = nullSafeList(caseData.getHearingOrdersBundlesDrafts()).stream()
-            .map(Element::getValue)
-            .filter(b -> Objects.equals(b.getHearingId(), caseData.getLastHearingOrderDraftsHearingId()))
-            .findFirst()
-            .map(HearingOrdersBundle::getOrders)
-            .map(ElementUtils::unwrapElements)
-            .orElse(emptyList());
+        final List<HearingOrder> orders = getOrders(caseData);
 
         if (isEmpty(orders)) {
             return;
         }
 
-        AbstractJudge judge = null;
-
-        if (hearing != null && hearing.getJudgeAndLegalAdvisor() != null
-            && isNotEmpty(hearing.getJudgeAndLegalAdvisor().getJudgeEmailAddress())) {
-            judge = hearing.getJudgeAndLegalAdvisor();
-        } else if (caseData.hasAllocatedJudgeEmail()) {
-            judge = caseData.getAllocatedJudge();
-        }
+        AbstractJudge judge = getAbstractJudge(caseData, hearing);
 
         if (judge == null || isEmpty(judge.getJudgeEmailAddress())) {
             return;
         }
 
-        final DraftOrdersUploadedTemplate content = contentProvider.buildContent(caseData, hearing, judge, orders);
+        final DraftOrdersUploadedTemplate content = draftOrdersContentProvider.buildContent(caseData, hearing, judge,
+            orders);
 
         notificationService.sendEmail(
             DRAFT_ORDERS_UPLOADED_NOTIFICATION_TEMPLATE,
             judge.getJudgeEmailAddress(),
             content,
             caseData.getId().toString()
         );
+    }
+
+    @Async
+    @EventListener
+    public void sendNotificationToAdmin(final DraftOrdersUploaded event) {
+        CaseData caseData = event.getCaseData();
+
+        final List<HearingOrder> orders = getOrders(caseData);
+
+        if (isEmpty(orders) || !orders.get(0).getType().equals(AGREED_CMO)) {
+            return;
+        }
+
+        final HearingBooking hearing = findElement(caseData.getLastHearingOrderDraftsHearingId(),
+            caseData.getHearingDetails())
+            .map(Element::getValue)
+            .orElse(null);
+
+        AbstractJudge judge = getAbstractJudge(caseData, hearing);
+
+        if (judge == null || isEmpty(judge.getJudgeEmailAddress())) {
```

Comment:
i think checking judge email is not needed here, should not be needed to sen email to admin